### ISSUE TITLE
[Critical] Function cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#92]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87551 |
| **Checkpoint** | 2.1 Keyboard Accessible › Keyboard Navigation (2.1.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 04 Work Items Workflow |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/work_items> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/05/26 at 3:28 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
Pattern: Each item in the Work Items table is not keyboard accessible as users cannot tab to them. See Work Items User Experience Video at 23:55.
Also occurs on Page 03 Objects and 06 Alerts

## Recommendation

Rule : 
Links, buttons, and interactive controls MUST be keyboard-focusable and operable.

How To Fix: 
For controls that navigate (link) to another location, fix this issue by using ONE of the following techniques:

1. Use a native HTML <a> element with an HREF attribute. An <a> element with an HREF comes with keyboard accessibility baked in.

<a href="application.html">Apply now!</a>

2. Ensure the control is:
a. Keyboard focusable using tabindex="0" AND
b. Keyboard operable using the Enter key in addition to mouse click.

NOTE: If you do not use a <a> and HREF, you will also need to add role="link" to ensure the link has a role to satisfy 4.1.2 Name, Role, Value, and a focus indicator will be needed to satisfy 2.4.7 Focus Visible.

<div tabindex="0" role="link" data-href="application.html">Apply now!</div>

Background: 
Some people cannot use a mouse due to vision or motor disabilities. Content that can be operated with a mouse must also be made operable with a keyboard. When content is operable through a keyboard, it becomes operable by a variety of assistive technologies such as speech input software, sip-and-puff software, on-screen keyboards, scanning software, and alternate keyboards.

## Source Code

```
<tr class="clickable" onclick="document.location.href=&quot;/work_items/show/63880&quot;">
        <td class="pl-5">
          <div class="is-flex is-align-items-center">
            <span class="action-icon mr-3">
              <span class="material-icons md-16" aria-hidden="true"><span>file_upload</span></span>
            </span>
            <span class="is-grey-dark">Ingest</span>
          </div>
        </td>
        <td class="wrap-long-words">Ablr-big-object.tar</td>

        

        <td class="is-grey-dark num text-sm">108.4 GB</td>
        <td class="is-grey-dark">Cleanup</td>
        <td><span class="badge is-success">Success</span></td>
        <td class="is-grey-dark text-sm is-uppercase">Feb  3, 2026</td>
      </tr>
```

## Screenshots

![04_keyboard](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87551/04_keyboard.png)
